src/main/java/com/google/sps/servlets/Recipe.java

```diff
@@ -25,7 +72,36 @@ public void setDescription(String givenDesc) {
     description = givenDesc;
   }
 
-  public void setId(int givenId) {
+  public void setDifficulty(String givenDiff) {
+    difficulty = givenDiff;
+  }
+
+  public void setPrepTime(String givenPTime) {
+    prepTime = givenPTime;
+  }
+
+  public void setCookTime(String givenCTime) {
+    cookTime = givenCTime;
+  }
```

Comment:
I think Java has types for representation of time intervals, you should look into that. Otherwise explain what the format should be.